### ISSUE TITLE
Runnable Jar

### ISSUE THREAD

**Issue by dtann80** [dtann80/ICS499_WordSearch#9]
Need to be able to build a runnable Jar from the project. Looks like the issue could be how resources are included with the Jar when creating it.

**Comment by timdan2** [comment]
This was a problem with how the special Gidugu font was being read (or NOT being read by the Jar).

Used this:
	public static String specialFontFile = "Gidugu.ttf";
	String fontFileName = Configuration.specialFontFile;
	InputStream is = this.getClass().getResourceAsStream(fontFileName);
	Font gidugu_font = Font.createFont(Font.TRUETYPE_FONT, is);

**Comment by timdan2** [comment]
Also, I learned that you can troubleshoot Jar file problems on Mac by opening Terminal and running this command:
java -jar [drag the problematic jar file here]

**Comment by wallgreeens** [comment]
I'm having some trouble opening a saved file from the gui.  I've made  a
method, but I'm not sure how the WordTrackerList works.  I'll try to keep
at it tomorrow, but I'll include what I've done.  I was able to create an
object that contains the id,  title, words, and the puzzle, but I'm not
sure how to load it into the gui.  If you go to puzzle.java and use my old
method of going through the words  on line 175 all the information will
print through the console otherwise it will print out incorrect
information.    I've commented out the wordtrackerlist on line 175.  when
you run the application it will ask you for an ID, which needs to be
there.  I've been using 008 and it will create an object based upon that
puzzle id.  sorry guys.  I'll keep trying tomorrow, sorry guys.  I worked
all day today and yesterday trying to figure this out.
